SingularityUI/app/components/requests/ScaleButton.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { Component, PropTypes } from 'react';
+
+import FormModal from '../common/FormModal';
+import Glyphicon from '../common/atomicDisplayItems/Glyphicon';
+
+export default class ScaleButton extends Component {
+
+  static propTypes = {
+    requestId: PropTypes.string.isRequired,
+    scaleAction: PropTypes.func.isRequired,
+    bounceAction: PropTypes.func.isRequired,
+    currentInstances: PropTypes.number
+  };
+
+  static INCREMENTAL_BOUNCE_VALUE = {
+    INCREMENTAL: {label: 'Kill old tasks as new tasks become healthy', value: true},
+    ALL: {label: 'Kill old tasks once ALL new tasks are healthy', value: false}
+  };
+
+  handleScale(data) {
+    console.log(data);
```

Comment:
I'm not going to delete these because I didn't add them. But @kwm4385 @wolfd here's a `console.log`. Was it there for a reason?